actionmailer/lib/action_mailer/delivery_methods.rb

```diff
@@ -31,7 +31,7 @@ module DeliveryMethods
 
       add_delivery_method :sendmail, Mail::Sendmail,
         location:  "/usr/sbin/sendmail",
-        arguments: "-i"
+        arguments: Gem::Version.new(Mail::VERSION.version) >= Gem::Version.new("2.8.0") ? %w[-i] : "-i"
```

Comment:
Maybe a comment pointing to the `mail` PR that changed this?

Reply:
Sure. Added. That commit ref is also in the commit message for this change.